src/resources/input.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
something is wrong with file structure, resources should be inside main package

Reply:
Done